sc-memory/sc-memory/sc_memory.hpp

```diff
@@ -962,11 +962,7 @@ class ScMemoryContext
       ScTemplateGenResult & result,
       ScTemplateParams const & params = ScTemplateParams::Empty,
       ScTemplateResultCode * resultCode = nullptr) noexcept(false);
-  SC_DEPRECATED(
-      0.8.0,
-      "Use callback-based ScMemoryContext::HelperSearchTemplate(ScTemplate const & templ, "
-      "ScTemplateSearchResultCallback const & callback, ScTemplateSearchResultCheckCallback const & checkCallback) "
-      "instead.")
+
```

Comment:
Write comment when to use this method and when to use callback-based ScMemoryContext::HelperSearchTemplate(ScTemplate const & templ, "
      "ScTemplateSearchResultCallback const & callback, ScTemplateSearchResultCheckCallback const & checkCallback) method